### PR TITLE
Add mutual TLS

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

**Comment by metacosm** [comment]
We need to rebase from upstream first. This fork looks out of date.

File: spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Istio - Spring Boot - Mutual TLS
+description: Booster presenting Istio Mutual TLS functionality
+source:
+  git:
+    url: https://github.com/snowdrop/spring-boot-istio-tls-booster.git
+    ref: automatic
```
Comment (metacosm) [inline spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml:6]: Is the latest code really in an `automatic` branch?

File: spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+metadata:
```
Comment (metacosm) [inline spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml:7]: I think this section needs to be removed.

**Review by metacosm — CHANGES_REQUESTED** [review]
Also, this new file needs to be in `current-community` instead of `1.5.10-community`. We also need an entry in `metadata.yml` to record the fact that a new mission is added.

**Comment by metacosm** [comment]
I'll perform that change since I had to force push changes. Sorry about that. Please review the comments anyway, they are still valid :)

**Comment by gytis** [comment]
Thanks @metacosm

File: spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml (hunk 1 above)
Comment (gytis) [reply spring-boot/1.5.10-community/istio-mutual-tls/booster.yaml:6]: I've copied it from the AB booster with an assumption that it's a new keyword used by the launcher which I'm not aware of...

**Pull request closed without merging**